src/Http/Controllers/NovaBreadcrumbsController.php

```diff
@@ -49,7 +49,7 @@ public function __invoke(Request $request)
         if ($pathParts->has(1)) {
             $this->resource = Nova::resourceForKey($pathParts->get(1));
 
-            if ($this->resource && ! $this->resource::breadcrumbs()) {
+            if ($this->resource && (! $this->resource::breadcrumbs() || ! method_exists($this->resource, 'breadcrumbs'))) {
```

Comment:
shouldn't it be the second one? check method availability, then call it ?

Reply:
you are right, i had it changed but didnt push